FBControlCoreTests/Tests/FBControlCoreLoggerTests.m

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "FBControlCoreLogger.h"
```

Comment:
Can you make this a Framework Import of `FBControlCore` to keep in line with the rest of the tests:

```
#import <FBControlCore/FBControlCore.h>
```